pyproject.toml

```diff
@@ -27,16 +28,18 @@ dependencies = [
     'graphviz>=0.13.2',
     "numpy>=1.21.0;python_version<'3.10'",
     "numpy>=1.23.3;python_version>='3.10' and python_version<'3.12'",
-    "numpy>=1.26.0;python_version>='3.12'",
+    "numpy>=1.26.0;python_version>='3.12' and python_version<'3.13'",
+    "numpy>=2.1.0;python_version>='3.13'",
     "pandas>=1.4.0;python_version<'3.11'",
     "pandas>=1.5.0;python_version>='3.11' and python_version<'3.12'",
-    "pandas>=2.1.1;python_version>='3.12'",
+    "pandas>=2.1.1;python_version>='3.12' and python_version<'3.13'",
+    "pandas>=2.2.3;python_version>='3.13'",
     'tqdm>=4.29',
-    'copulas>=0.12.0',
-    'ctgan>=0.10.2',
-    'deepecho>=0.6.1',
-    'rdt>=1.13.2',
-    'sdmetrics>=0.17.0',
+    'copulas>=0.12.1',
+    'ctgan @ git+https://github.com/sdv-dev/CTGAN@main',
```

Comment:
can you point this to the release candidate to test it? https://pypi.org/project/ctgan/0.11.0.dev0/

Reply:
@amontanez24 done

Reply:
@rwedge The actual release is out now!